qutebrowser/mainwindow/statusbar/zoom.py

```diff
@@ -0,0 +1,28 @@
+"""Zoom percentage displayed in the statusbar."""
+
+from qutebrowser.browser import browsertab
+from qutebrowser.mainwindow.statusbar import textbase
+from qutebrowser.qt.core import pyqtSlot, QObject
+from qutebrowser.config import config
+
+
+class Zoom(textbase.TextBase):
+
+    """Shows zoom percentage in current tab."""
+
+    def __init__(self, parent: QObject = None) -> None:
+        super().__init__(parent)
+        self.on_zoom_changed(1)
+
+    @pyqtSlot(float)
+    def on_zoom_changed(self, factor: float) -> None:
+        """Update percentage when factor changed."""
+        if factor == 1 and config.val.statusbar.zoom.show == 'non-default':
+            self.setText("")
+            return
+        percentage = int(100 * factor)
```

Comment:
For bigger zoom levels, this seems to disagree with the configured zoom (and the one displayed in the message):

<img width="621" height="80" alt="Image" src="https://github.com/user-attachments/assets/173be9f1-7d91-4ffe-ad8f-dfbc9cd1ae5c" />

<img width="628" height="80" alt="Image" src="https://github.com/user-attachments/assets/d3fa4705-32a7-484c-9771-1266e292b9ee" />

Looks like `factor` is `3.9999999999999996` and `4.999999999999999`, probably due to [floating point numbers being annoying](https://0.30000000000000004.com/).

I suppose a proper fix would be to somehow refactor the actual zoom logic to use e.g. an int percentage instead, but in the meantime I'm perfectly fine with just going for


```suggestion
        percentage = round(100 * factor)
```

(and adding a test for it) instead. What do you think?